### ISSUE TITLE
Formatting is removed on save

### ISSUE THREAD

**Issue by zaytri** [kawal7415/excel#29]
**Before:**
<img width="221" alt="Screen Shot 2020-06-15 at 1 31 28 PM" src="https://user-images.githubusercontent.com/9072119/84703344-f601c180-af0c-11ea-8200-283663f45880.png">

**After:**
<img width="235" alt="Screen Shot 2020-06-15 at 1 31 07 PM" src="https://user-images.githubusercontent.com/9072119/84703368-fd28cf80-af0c-11ea-9b8b-c82031189aeb.png">

**Code:**
```dart
import 'dart:io';
import 'package:path/path.dart';
import 'package:excel/excel.dart';

final String FILE_NAME = 'Book1.xlsx';
final String SHEET_NAME = 'Sheet1';
final File file = File(join(FILE_NAME));
final Excel excel = Excel.decodeBytes(file.readAsBytesSync());

void main(List<String> args) async {
  save();
}

void save() {
  excel.encode().then((onValue) {
    File(join('output.xlsx'))
    ..writeAsBytesSync(onValue);
  });
}
```

**Attachments:**
[Book1.xlsx](https://github.com/kawal7415/excel/files/4782851/Book1.xlsx)

**Environment:**
Dart VM version: 2.8.1 (stable) (Thu Apr 30 09:25:21 2020 +0200) on "macos_x64"
Excel package version: 1.0.9
MacOS version: 10.15.5

**Comment by ialameh** [comment]
you may need to turn the save into a future and await for the save

```
void main(List<String> args) async {
  await save();
}

Future<void> save() async{
  await excel.encode().then((onValue) {
    File(join('output.xlsx'))
    ..writeAsBytesSync(onValue);
  });
}
```

**Comment by zaytri** [comment]
@ialameh unfortunately that does not solve the issue.

**Comment by kawal7415** [comment]
We have fixed the issue and it will be available in version `excel: ^1.1.0` by tomorrow EOD.
